src/test/kotlin/com/atlassian/performance/tools/sshubuntu/api/ssh/SshUbuntuTest.kt

```diff
@@ -73,4 +76,14 @@ class SshUbuntuTest {
 
         assertThat(privileged).isTrue()
     }
+
+    @Test
+    fun shouldAptGetWork() {
+        val result = execute(
+            cmd = "export DEBIAN_FRONTEND=noninteractive; apt-get update && apt-get install software-properties-common -y -qq",
+            timeout = Duration.ofMinutes(3)
+        )
+
+        assertThat(result.isSuccessful()).isTrue()
```

Comment:
This assertion will never fail, because `SshConnection.execute` already does the check. Use `SshConnection.safeExecute` instead.